cypress/integration/table/search.spec.ts

```diff
@@ -646,7 +646,7 @@ describe('Search', () => {
         '2022-01-01 00:00:00'
       );
       cy.get('input[aria-label="to, date-time input"]').type(
-        '2022-01-02 00:00:00'
+        '2022-01-02 00:00:59'
```

Comment:
Again, should probably just type `00:00` instead of specifying the seconds to mirror expected user behaviour

Reply:
I've changed it in the unit and e2e tests in the latest commit